hedera-mirror-datagenerator/src/main/java/com/hedera/datagenerator/sdk/supplier/token/TokenWipeTransactionSupplier.java

```diff
@@ -0,0 +1,56 @@
+package com.hedera.datagenerator.sdk.supplier.token;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2020 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import java.time.Instant;
+import lombok.Builder;
+import lombok.Value;
+
+import com.hedera.datagenerator.common.Utility;
+import com.hedera.datagenerator.sdk.supplier.TransactionSupplier;
+import com.hedera.hashgraph.sdk.account.AccountId;
+import com.hedera.hashgraph.sdk.token.TokenId;
+import com.hedera.hashgraph.sdk.token.TokenWipeTransaction;
+
+@Builder
+@Value
+public class TokenWipeTransactionSupplier implements TransactionSupplier<TokenWipeTransaction> {
+    private final String accountId;
+    private final String tokenId;
+
+    //Optional
+    @Builder.Default
+    private final long amount = 1;
+
+    @Builder.Default
+    private final long maxTransactionFee = 1_000_000_000;
+
+    @Override
+    public TokenWipeTransaction get() {
+        return new TokenWipeTransaction()
+                .setAccountId(AccountId.fromString(accountId))
+                .setAmount(amount)
+                .setMaxTransactionFee(maxTransactionFee)
+                .setTokenId(TokenId.fromString(tokenId))
+                .setTransactionMemo(Utility.getEncodedTimestamp() + "_Mirror node wiped test token at " + Instant
```

Comment:
I realize we're repeating the encodedTimestamp + message + readable instant time.
We could move this into the new Utility class you added and make it easier

```java
    public static String getTransactionMemo(String message) {
         return Utility.getEncodedTimestamp() + "_" + message + " at " + Instant.now();
     }
```

```suggestion
                .setTransactionMemo(Utility.getTransactionMemo("Mirror node wiped test token"))
```

Reply:
Good call, did this for all classes.